app/code/Magento/Customer/view/frontend/web/template/show-password.html

```diff
@@ -0,0 +1,9 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+ -->
+
+<input type="checkbox" name="show-password" title="Show Password" id="show-password" class="checkbox" data-role="show-password" data-bind="checked: isChecked">
```

Comment:
It's better to use Magento bindings syntax.
```suggestion
<input type="checkbox" name="show-password" title="Show Password" id="show-password" class="checkbox" data-role="show-password" ko-checked="isChecked">
```

Reply:
Magento bindings used instead of ko binding. code is updated.